src/domd_cfg_spider.c

```diff
@@ -0,0 +1,293 @@
+/*
+ * Copyright (c) 2023 EPAM Systems
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <domain.h>
+#include <string.h>
+#include <zephyr/xen/public/domctl.h>
+
+static char* domd_dtdevs[] = {
+    "/soc/dma-controller@e7350000",
+    "/soc/dma-controller@e7351000",
+    "/soc/mmc@ee140000",
```

Comment:
(note) I'm not sure that this is correct, at far as I know we should delete xen pass through property for MMC and I thought that domd doesn't able to work with this device after it

Reply:
dropped mmc node from the config